test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -14,30 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 registryItems:
   applications:
-    - {webflux-projectB-scenario: nq 0}
-    - {webflux-projectA-scenario: nq 0}
+    - {webflux-scenario: nq 0}
```

Comment:
Why do you remove the 2nd app? It makes this test case can't verify the ref. I think you need to add it back.